raiden/messages.py

```diff
@@ -1662,6 +1662,39 @@ def unpack(
         )
         return monitoring_request
 
+    def verify(self, partner_address, requesting_address):
```

Comment:
add types in args and return value. And if this is function to verify integrity of a 'RequestMonitoring` message please rename the function accordingly: `verify_request_monitoring()` or something similar.